src/lib.rs

```diff
@@ -14,6 +14,7 @@ pub mod file_traits;
 pub mod seek_hole;
 pub mod signal;
 pub mod sock_ctrl_msg;
+pub mod struct_util;
```

Comment:
As far as I know, this is only needed by the Firecracker kernel loader crate. So this is a case of exporting an API that only has one consumer.
I would prefer that Firecracker either a) define the `struct_util` API in its own kernel loader or b) switch to the rust-vmm `linux-loader` crate. But I don't think we should add it to this rust-vmm crate unless there is more than one user.

Reply:
I'm OK with both solutions:)

Reply:
@jiangliu Sweet :)

Could you also fix the unit test and clippy warnings?

Reply:
I'm trying to access the buildbike logs:)

Reply:
> I'm trying to access the buildbike logs:)

You should be able to see it now.

Reply:
@sameo it seems like struct_util is also used by linux-loader. Only that instead of using it from vmm-sys-utils the utilities are implemented directly in the linux-loader crate.

I checked the Firecracker code and the only place where we are using the struct_util is in the kernel-loader so if you think it makes more sense there and no other crate can benefit from it we can also just leave it there.

Reply:
@andreeaflorescu I think it makes more sense to keep it in the single crate that consumes it, yes.